### ISSUE TITLE
[BUG] Errors when opening file in Excel after saving a file with conditional formatting

### ISSUE THREAD

**Issue by glen-84** [exceljs/exceljs#1305]
## 🐛 Bug Report

Lib version: 3.9.0

## Steps To Reproduce

```javascript
const Excel = require("exceljs");
const fs = require("fs");

const inputFilePath = "input.xlsx";
const outputFilePath = "output.xlsx";

(async () => {
    const workbook = new Excel.Workbook();

    const data = await fs.promises.readFile(inputFilePath);

    await workbook.xlsx.load(data.buffer);

    // Write.
    await fs.promises.writeFile(outputFilePath, await workbook.xlsx.writeBuffer());
})().catch((e) => console.error(e));
```

[input.xlsx](https://github.com/exceljs/exceljs/files/4689864/input.xlsx)
[output.xlsx](https://github.com/exceljs/exceljs/files/4689865/output.xlsx)

Opening the output file with Excel results in the following:

![image](https://user-images.githubusercontent.com/261509/83035404-b2b4e100-a039-11ea-9a1a-db08939a7f4c.png)

```xml
<? version="1.0" encoding="UTF-8" standalone="yes"?>
<recoveryLog xmlns="http://schemas.openxmlformats.org/spreadsheetml/2006/main">
  <logFileName>
    error043120_01.xml
  </logFileName>
  <summary>
    Errors were detected in file 'C:\Users\Rob\Documents\glen-tmp\output.xlsx'
  </summary>
  <removedParts summary="Following is a list of removed parts:">
    <removedPart>
      Removed Part: /xl/styles.xml part with XML error.  (Styles) Load error. Line 2, column 6585.
    </removedPart>
  </removedParts>
  <repairedRecords summary="Following is a list of repairs:">
    <repairedRecord>
      Repaired Records: Cell information from /xl/worksheets/sheet2.xml part
    </repairedRecord>
    <repairedRecord>
      Repaired Records: Column information from /xl/worksheets/sheet2.xml part
    </repairedRecord>
    <repairedRecord>
      Repaired Records: Conditional formatting from /xl/worksheets/sheet2.xml part
    </repairedRecord>
  </repairedRecords>
</recoveryLog>
```

Just opening and saving the file causes the file size to go from 33 KB to 14 KB. 😮

If I clear the conditional formatting in the input file, then I don't see the errors (although the file size still drops dramatically, which is a bit worrying).

## The expected behaviour:

No errors when opening the file in Excel, and possibly more similar file size of input and output files.

## Possible solution (optional, but very helpful):

n/a

---

**Edit:** Here's a smaller reproduction:

[simple3-input.xlsx](https://github.com/exceljs/exceljs/files/4690423/simple3-input.xlsx)
[simple3-output.xlsx](https://github.com/exceljs/exceljs/files/4690441/simple3-output.xlsx)

**Comment by aliaksei-lebedzeu** [comment]
Have got the same issue. After a bit of investigation found that the rule value is broken.
So in the template:
![image](https://user-images.githubusercontent.com/67019510/84810038-c5875980-afd8-11ea-98fe-c670dfc6f432.png)
But, once the doc is saved the following gets saved even if no changes have been made before that:
![image](https://user-images.githubusercontent.com/67019510/84810219-067f6e00-afd9-11ea-877f-b4879f6c2ad2.png)

**Comment by cedardray** [comment]
Hi, 
I've been facing the same issue, it turns out the render in dxf-xform.js is rendering elements in the wrong order. Restoring the order to the one specified here [http://www.datypic.com/sc/ooxml/e-ssml_dxf-1.html](http://www.datypic.com/sc/ooxml/e-ssml_dxf-1.html) fixed the issue for me.

**Comment by DanilShapilov** [comment]
if you, like me, came here because of this bug and don't need conditional formatting, use this command on a worksheet:
`worksheet.removeConditionalFormatting();`

**Comment by Siemienik** [comment]
Fixed in  #1574
